Dockerfile

```diff
@@ -30,7 +30,6 @@ COPY .ruby-version Gemfile Gemfile.lock ./
 
 # Install gems and remove gem cache
 RUN bundler -v && \
-    bundle config set no-cache 'true' && \
```

Comment:
should this also be reverted, this is a bundler command.

Reply:
yes, now reverted.